README.md

```diff
@@ -1,32 +1,96 @@
- # TerraGov Marine Corps
-[![Percentage of issues still open](https://isitmaintained.com/badge/open/tgstation/TerraGov-Marine-Corps.svg)](https://isitmaintained.com/project/tgstation/TerraGov-Marine-Corps "Percentage of issues still open") [![Average time to resolve an issue](https://isitmaintained.com/badge/resolution/tgstation/TerraGov-Marine-Corps.svg)](https://isitmaintained.com/project/tgstation/TerraGov-Marine-Corps "Average time to resolve an issue")
 
- [![forthebadge](https://forthebadge.com/images/badges/built-with-resentment.svg)](https://forthebadge.com) [![forthebadge](https://forthebadge.com/images/badges/contains-technical-debt.svg)](https://forthebadge.com) [![forinfinityandbyond](https://user-images.githubusercontent.com/5211576/29499758-4efff304-85e6-11e7-8267-62919c3688a9.gif)](https://www.reddit.com/r/SS13/comments/5oplxp/what_is_the_main_problem_with_byond_as_an_engine/dclbu1a)
+<h1>
 
- This is a fork based off the July-2018 version of ColonialMarines. To see the original, GPL repo, go [here](https://github.com/MrStonedOne/cmhistory)
+TerraGov Marine Corps 
+[![Badge Issues]][Issues]
+[![Badge Time]][Time]
 
-Code and other contributions are licensed under AGPL unless otherwise specified below
+</h1>
 
-Artwork added after -this commit- is CC-BY-NC 3.0
+<div align = center>
 
-Sprites added in the PR https://github.com/tgstation/TerraGov-Marine-Corps/pull/367 are licensed under CC-BY-NC 3.0
-See commit 36b8db4952be79a79b28a6738889d9e9eb23b12a
+![Badge Resentment]![Badge Dept][![Badge Byond]][Byond]
 
-Sprites added in commit d407e97e26ee5e6bb1daf945a8eb3bd9a6b11976 are CC-BY-NC 3.0
+</div>
 
-All sounds added/remixed in the PRs:
-https://github.com/tgstation/TerraGov-Marine-Corps/pull/5261
-https://github.com/tgstation/TerraGov-Marine-Corps/pull/5861
-are licensed under CC-BY-NC-SA 3.0
+<br>
 
-All icons in the PR:
-https://github.com/tgstation/TerraGov-Marine-Corps/pull/6010
-are licensed under CC-BY-NC-SA 3.0
+## Licensing
 
-Rouny Sprite in the PR:
-https://github.com/tgstation/TerraGov-Marine-Corps/pull/6938
-are licensed under CC BY-SA 3.0
+This fork is based on the `July 2018` version of **[ColonialMarines]** .
 
-Sound added in PR https://github.com/tgstation/TerraGov-Marine-Corps/pull/5604 is taken from https://freesound.org/people/nicStage/sounds/127731/ (CC Attribution)
+<br>
+<br>
 
-The TGS DMAPI API is licensed as a subproject under the MIT license.
+<div align = center>
+
+*Contributions, if not otherwise specified,* <br>
+*are licensed under the **[AGPL][License AGPL]** license.*
+
+<br>
+<br>
+
+What            | From                                 | License
+:--------------:|:------------------------------------:|:------------------------:
+`Artwork`       | *after -this commit-*                | [![Badge CC-BY-NC]][License CC-BY-NC]
```

Comment:
Please add that this means https://github.com/tgstation/TerraGov-Marine-Corps/commit/92ee94da6e144d4420c1ec4f83a4ee785d61dc9b